packages/ListBox/stories/examples/Showcase.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean, text } from "@storybook/addon-knobs";
+import { Rule } from "storybook/assets/styles/common.styles";
+import * as characters from "../fixtures/characters";
+import ListBox from "../../src";
+
+const getKnobs = () => ({
+  hasImplicitAll: boolean("hasImplicitAll", false),
```

Comment:
Does this have an effect in the Showcase? If we can't see the difference (even in the `actions` panel) then we should just omit it and explain it in the README, and/or MDX and/or Examples stories.

Reply:
I'll remove it for now and include it somewhere else in a future PR